tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html

```diff
@@ -196,7 +196,7 @@
       observers: ['_optionsOrSessionGroupsChanged(options.*, sessionGroups.*)'],
 
       _optionsOrSessionGroupsChanged() {
-        if (this.options === null) {
```

Comment:
FWIW, the general rule that I’m familiar with is “always use `===`
except specifically for `== null`, because it is the same\* as
`=== null || === undefined`, which is easy to understand and often what
you want”. This is common enough to have an [ESLint option][esl], and is
also [called out in the Google style guide][gs].

\* Except specifically that `document.any == null` because we simply
cannot have nice things.

[esl]: https://eslint.org/docs/rules/eqeqeq#allow-null
[gs]: https://google.github.io/styleguide/jsguide.html#features-equality-checks-exceptions

Reply:
At some point in the sequence of state changes, `this.options` became `undefined`, not `null`.  This caused `this.options.configuration` (immediately below) to choke.  I didn't track down the root cause, but just figured that we should treat `undefined` the same anyway, and that seemed to work.  Indeed, `!this.options` works too.